src/cli/commands/types.py

```
@@ -535,7 +535,11 @@ def generate_class_code(name: str, spec_data: dict) -> list[str]:
                     progress.advance(task)
             elif spec is not None:
                 # 単一型仕様
-                code_lines.extend(generate_class_code("GeneratedType", spec.model_dump()))
+                if isinstance(spec, str):
+                    # 参照文字列の場合はスキップ（未解決の参照）
```

Ruff (0.14.0)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)